src/coredump/meson.build

```diff
@@ -23,5 +23,5 @@ tests += [
         [['src/coredump/test-coredump-vacuum.c',
           'src/coredump/coredump-vacuum.c',
           'src/coredump/coredump-vacuum.h'],
-         [], [], [], 'ENABLE_COREDUMP', 'manual'],
+         [], [], [], '', 'manual'],
 ]
```

Comment:
The assumption for not building test cases when a feature is disabled was that this makes builds faster, and possibly avoids pulling in dependencies just to build tests. It seems that in this case the extra build time is small, and that no big dependencies are pulled in. The only potential downside is that people use `-Dinstall-tests=true` and turn of some features, they might unexpectedly get additional tests that don't make sense for them installed. But that should be fairly niche — I think it's fair to assume that installed tests are mostly used by big distros which enable all possible features. Seems OK to do this.